packages/react-components/react-carousel-preview/library/src/components/useEmblaCarousel.ts

```diff
@@ -39,6 +40,17 @@ export function useEmblaCarousel(
   });
   const emblaApi = React.useRef<EmblaCarouselType | null>(null);
 
+  const autoplayRef = React.useRef<boolean>(false);
+  /* Our autoplay button, which is required by standards for autoplay to be enabled, will handle controlled state */
+  const enableAutoplay = (autoplay: boolean) => {
+    autoplayRef.current = autoplay;
+    if (autoplay) {
+      emblaApi.current?.plugins().autoplay.play();
+    } else {
+      emblaApi.current?.plugins().autoplay.stop();
+    }
+  };
```

Comment:
```suggestion
  const enableAutoplay = React.useCallback((autoplay: boolean) => {
    autoplayRef.current = autoplay;
    if (autoplay) {
      emblaApi.current?.plugins().autoplay.play();
    } else {
      emblaApi.current?.plugins().autoplay.stop();
    }
  }, []);
```

Let's memoize to avoid the context explosion